### PR TITLE
docs: Cypress App Notifications

### PR DESCRIPTION
**Pull request by lmiller1990**
Closes https://github.com/cypress-io/cypress-documentation/issues/5280

Some docs & tips for the new In App Notifications feature.

We've got something about Debug in https://deploy-preview-5327--cypress-docs.netlify.app/guides/cloud/recorded-runs#Debug but it's pretty out of the way and hard to find in general - I think we want to put all the App <> Cloud features together.

This whole part of the documentation isn't very cohesive, but doing an overhaul is definitely out of the scope right now.

### CODE REVIEW

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+Then turn on notifications for Cypress:
+
+<DocsImage
+  src="/img/guides/cloud/app-integrations/macos-2.png"
+  alt="Enable Cypress Notifications"
+/>
+
+### Linux
+
+How your desktop manages notifications depends on your Linux distribution your windows manager.
```
Comment (lmiller1990) [inline docs/guides/cloud/cypress-app-integration.mdx:77]: I spend quite a while trying to understand how Ubuntu (Gnome) manages notifications for desktop applications not installed via Software Center. I gave up after about an hour, I can't figure it out. It uses something called `gsettings`, but the manual doesn't include any examples and all the things I tried didn't work.

Either way, out of the box every seems to work great - if a user does get stuck, we can help them out. Most non power users are going to be Ubuntu or other "easy" distributions, that just work - power users will likely know their OS better than we do and will be able to figure it out.

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  alt="Sending a test notification"
+/>
+
+If this is not working as expected, see [Troubleshooting](#Troubleshooting) below.
+
+## Troubleshooting
+
+Notifications and the associated permissions work slightly differently on each operating system.
+
+### MacOS
```
Comment (MikeMcC399) [inline docs/guides/cloud/cypress-app-integration.mdx:59]:
```suggestion
### macOS
```

according to Apple on https://support.apple.com/macos

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+If notifications are not showing up in MacOS, under "System Preferences" select "Notifications" and choose "Cypress":
```
Comment (MikeMcC399) [inline docs/guides/cloud/cypress-app-integration.mdx:61]:
```suggestion
If notifications are not showing up in macOS, under "System Preferences" select "Notifications" and choose "Cypress":
```

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+<DocsImage
+  src="/img/guides/cloud/app-integrations/macos-1.png"
+  alt="Notification in System Preferences for MacOS"
```
Comment (MikeMcC399) [inline docs/guides/cloud/cypress-app-integration.mdx:65]:
```suggestion
  alt="Notification in System Preferences for macOS"
```

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+### Linux
+
+How your desktop manages notifications depends on your Linux distribution your windows manager. 
+If you are not seeing notifications when you think you should be, 
+please file an issue including the operating system and version.
+
+### Windows
+
+Notifications for Windows is currently not supported. Supported will be added in a later release.
```
Comment (MikeMcC399) [inline docs/guides/cloud/cypress-app-integration.mdx:83]:
```suggestion
Notifications for Windows is currently not supported. Support will be added in a later release.
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Typo suggestions added in-line.

**Comment by lmiller1990** [comment]
> Typo suggestions added in-line.

That was fast, thank you! 💯

**Comment by MikeMcC399** [comment]
@lmiller1990 

You're welcome! You also need to run linting.

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+Then turn on notifications for Cypress:
+
+<DocsImage
+  src="/img/guides/cloud/app-integrations/macos-2.png"
+  alt="Enable Cypress Notifications"
+/>
+
+### Linux
+
+How your desktop manages notifications depends on your Linux distribution your windows manager.
```
Comment (MikeMcC399) [inline docs/guides/cloud/cypress-app-integration.mdx:77]: Not sure about this sentence:
"How your desktop manages notifications depends on the windows manager of your Linux distribution"
perhaps?

File: docs/guides/cloud/cypress-app-integration.mdx [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+If you are not seeing notifications when you think you should be,
+please file an issue including the operating system and version.
```
Comment (jordanpowell88) [inline docs/guides/cloud/cypress-app-integration.mdx:79]: We should put a link here to the issue.... I'm assuming we want this one https://github.com/cypress-io/cypress/issues/new/choose

**Review by jordanpowell88 — CHANGES_REQUESTED** [review]
Just a few small fixes. Overall good job outlining this with quality screenshots

**Comment by warrensplayer** [comment]
The work in this PR is being considered if it will be incorporated into the work done with this PR: https://github.com/cypress-io/cypress-documentation/pull/5348

**Comment by lmiller1990** [comment]
We've got docs live already for this: https://docs.cypress.io/guides/cloud/recorded-runs#Notifications

I think this is fine for now. If we have people asking questions or getting confused, we can definitely revisit and update them.

**Pull request closed without merging**